### DISCUSSION TITLE
Which is the best widget for showing images (e.g. PNGs)?

### DISCUSSION THREAD

**Discussion by mark-summerfield** [fltk-rs/fltk-rs#1344] [Q&A] [answered]
I want to show images in an app I'm developing. The images will be PNGs. I want the user to be able to scroll if the image is bigger than the widget & to be able to select regions by drawing rects, as well as identifying points by clicking. Is Frame the right widget for these purposes?

**Comment by MoAlyousef** [comment, marked as answer]
A Frame would be the most appropriate. If you need the scroll capabilities, you would have to place the frame inside a Scroll widget, resize the frame to the size of the image, and the Scroll will automatically show scrollbars if a widget inside it is drawn beyond its borders.
